portfolio/src/main/webapp/script.js

```diff
@@ -14,6 +14,9 @@
 
 async function getData() {
     const response = await fetch('/data');
-    const data = await response.text();
-    document.getElementById('data-container').innerHTML = data;
+    const comments = await response.json();
+    let html = "<br>";
```

Comment:
Can you explain a bit what format you're trying to get your comments into? I think you might be able to achieve something similar using `<p>` tags instead.

Reply:
I was just more focused on getting something out onto the screen because the main focus of the tutorial is the json parsing and formatting. I realize now using `<p>` tags is probably a better idea, thanks!